pkg/extender/extender.go

```diff
@@ -265,6 +276,91 @@ func (e *Extender) processFilterRequest(w http.ResponseWriter, req *http.Request
 	}
 }
 
+func (e *Extender) isNodeHyperConverged() error {
+	fn := func(object runtime.Object) error {
```

Comment:
Can you add unit tests for this function? You should be able to create pods and query the metrics endpoints for the stats